__tests__/get-endpoints.test.js

```diff
@@ -47,4 +48,40 @@ test("handles invalid endpoints", () => {
         });
 });
 
+describe("GET /api/articles/:id", () => {
+    it("responds with an article for a valid ID", () => {
+        let expectedArticle = testData.articleData[0];
```

Comment:
We can't use the test data in full to check that the data is correct. The test data is the data before it's seeded and inserted into the database so it's missing an article_id and I can see you've done some manipulation below to change the date which also changes during the seed process. 

The reason this is passing is because .toMatchObject only performs a partial match, it doesn't care if there's added properties that you don't declare, it only cares about matching the properties you have named. 

I guess the quickest 'fix' to this is to add article_id to your expectedArticle, but have a look at result.body to see if there's anything else different to expectedArticle